packages/migrate/migrations/routes/utils.js

```diff
@@ -176,13 +176,13 @@ export function get_exports(node) {
 		} else if (
 			ts.isFunctionDeclaration(statement) &&
 			statement.name &&
-			statement.modifiers?.[0]?.kind === ts.SyntaxKind.ExportKeyword
+			ts.getModifiers(statement)?.[0]?.kind === ts.SyntaxKind.ExportKeyword
```

Comment:
I'm wondering why you checked `ts.canHaveModifiers` in the other two files but not this one

Reply:
Because the statement node already is marked as being able to have modifiers.